.env.template

```diff
@@ -0,0 +1,22 @@
+# To get started, copy this file to .env and set your beacon http endpoint
+
+BLOB_ARCHIVER_L1_BEACON_HTTP=<unset>
+BLOB_ARCHIVER_DATA_STORE=s3
+BLOB_ARCHIVER_S3_ENDPOINT=172.17.0.1:9000
+BLOB_ARCHIVER_S3_ACCESS_KEY=admin
+BLOB_ARCHIVER_S3_SECRET_ACCESS_KEY=password
+BLOB_ARCHIVER_S3_ENDPOINT_HTTPS=false
+BLOB_ARCHIVER_S3_BUCKET=blobs
+BLOB_ARCHIVER_METRICS_ENABLED=true
+BLOB_ARCHIVER_METRICS_PORT=7300
+BLOB_ARCHIVER_ORIGIN_BLOCK=0x0
```

Comment:
I see flags like LISTEN_ADDRESS and POLL_INTERVAL that are not listed here, should they be?
Also should listen_address be a common flag instead of definted independently in both services?

Reply:
> LISTEN_ADDRESS and POLL_INTERVAL that are not listed here, should they be

They have default values (of `0.0.0.0:8000` and `6s`), which are probably reasonable for local dev and prod -- which is why they're excluded here. Not opposed to enumerating all the configs if you think it's clearer?

> listen_address be a common flag instead of defined independently in both services

I don't think so -- we probably want these to be defined separately so you can choose to run them on different ports/addresses. e.g. you could run the archiver on 127.0.0.1:8000 and the api on 0.0.0.0:8001

Reply:
No not clearer, just wasn't sure what your criteria was for putting some here vs not.

Reply:
My goal is to have a file that mostly (w/ the exception of URL + origin block) can be copy/pasted for local dev.